### PR TITLE
Update man page for fi_pingpong.

### PR DESCRIPTION
**Pull request by bturrubiates**
@jsquyres @goodell @shantonu @j-xiong 

Signed-off-by: Ben Turrubiates <bturrubi@cisco.com>

### CODE REVIEW

File: man/fi_pingpong.1.md [hunk 1]
```diff
@@ -57,29 +69,35 @@ The client's command line options must match those used on the server. If they d
...
+## Fabric Filtering
 
 *-p \<provider_name\>*
-: The name of the underlying fabric provider (e.g., sockets, psm, usnic, etc.). If a provider is not specified via the -f switch, the test will pick one from the list of available providers (as returned by fi_getinfo(3)).
+: The name of the underlying fabric provider (e.g., sockets, psm, usnic, etc.).
+  If a provider is not specified via the -f switch, the test will pick one from
+  the list of available providers (as returned by fi_getinfo(3)).
 
-*-p \<endpoint\>* where endpoint = (dgram|rdm|msg)
+*-p \<endpoint\>*
```
Comment (shantonu) [inline man/fi_pingpong.1.md]: Should be ```-e```

Comment (bturrubiates) [inline man/fi_pingpong.1.md]: > Should be -e

Thanks! Good catch, fixed.

File: man/fi_pingpong.1.md [hunk 2]
```diff
@@ -57,29 +69,35 @@ The client's command line options must match those used on the server. If they d
   provided then the client will connect to 47592 by default. The server ignores
   this option.
 
-## Fabric
+## Fabric Filtering
 
 *-p \<provider_name\>*
-: The name of the underlying fabric provider (e.g., sockets, psm, usnic, etc.). If a provider is not specified via the -f switch, the test will pick one from the list of available providers (as returned by fi_getinfo(3)).
+: The name of the underlying fabric provider (e.g., sockets, psm, usnic, etc.).
+  If a provider is not specified via the -f switch, the test will pick one from
```
Comment (j-xiong) [inline man/fi_pingpong.1.md]: -p here?

File: man/fi_pingpong.1.md [hunk 3]
```diff
@@ -125,31 +143,36 @@ Specifically, this will run a pingpong test with:
...
+switch, the test will pick one from the list of available providers (as
+returned by fi_getinfo(3)).
 
 If no endpoint type is specified, 'dgram' is used.
 
-The default tested sizes are:  64, 256, 1024, 4096.
-
-If no server address is specified, the server address is determined by the selected provider. With the current implementation of libfabric, it means that the picked address will be the first address in the list of available addresses matching the selected provider.
-
+The default tested sizes are:  64, 256, 1024, 4096, 655616, and 1048576. The
```
Comment (bturrubiates) [inline man/fi_pingpong.1.md]: > 655616 is a weird size, is it a typo?

You know, I just copied that from the source for the test. It is a typo though, that should be `2 << 15` or `65536`. I'll fix that and also update the test.

**Comment by j-xiong** [comment]
👍

**Pull request merged**